widgets/BucketSinging.js

```diff
@@ -223,10 +223,13 @@ async function initContext(){
     }
 
     div.empty();
-    div.append(`<div>This event will have group singing! To partcipate, you need to use <b>wired</b> headphones, and we will do some calibration.<br><br>You should <b>skip this</b> if:<ul>
+    div.append(`<div>This event will have group singing! To partcipate, you need to use <b>wired</b> headphones, and we will do some calibration.<br><br>
+                  During the singing part of the ritual, you will be randomly assigned to one of 4 buckets. You can choose a different bucket if you like. You will be able to hear the voices of everyone ahead of you. 
+                  If you are in the last bucket, nobody will hear you sing. Our group singing software automatically makes a recording per song that includes the voices of everyone in every bucket, including the last one.
```

Comment:
Maybe "nobody will hear you sing live", since people listening to the recording later will hear you?